src/utils/near-wallet.js

```diff
@@ -52,7 +52,7 @@ export class Wallet {
     // Omitting the accountId will result in the user being
     // asked to sign all transactions.
     this.createAccessKeyFor = createAccessKeyFor;
-    this.network = 'testnet';
+    this.network = process.env.REACT_APP_ENV === 'dev' ? 'testnet' : 'mainnet';
```

Comment:
why do we need testnet now, if we are moving to prod, then irrespective of local or prod, mainnet should be used
If we (devs) need to check something, we can manually do it
we should add a config file for such things (to carry all types of contract)
for eg: https://github.com/alpha-fi/cheddar-tic-tac-toe-ui/blob/master/src/near/config.ts

Reply:
This isn't a contract it's just a config switching mainnet and testnet , and manually changing the config isn't a good practice with the current approach you can just switch the env variable and every credential changes from dev to prod

Reply:
but there are the changes related to the network config right?

Reply:
resolved it to the contract-addresses file